.pfnci/hint.pbtxt

```diff
@@ -3,22 +3,20 @@
 # Proto type: xpytest.proto.HintFile
 # https://github.com/chainer/xpytest/blob/master/proto/test_case.proto
 
-# Too flaky tests.  Retry up to 4 times.
-rules { name: "chainerx_tests/unit_tests/routines_tests/test_arithmetic.py" xdist: 8 deadline: 240 retry: 4 }
-rules { name: "chainerx_tests/unit_tests/routines_tests/test_math.py" xdist: 8 deadline: 240 retry: 4 }
-rules { name: "chainer_tests/functions_tests/activation_tests/test_elu.py" xdist: 2 retry: 4 }
-
 # Slow tests take 60+ seconds.
 rules { name: "chainer_tests/functions_tests/activation_tests/test_crelu.py" xdist: 4 deadline: 240 }
+rules { name: "chainer_tests/functions_tests/activation_tests/test_elu.py" xdist: 2 }
```

Comment:
Shouldn't this entry be removed?

Reply:
> Shouldn't this entry be removed?

Is the test fast enough?

Reply:
I don't know, but this entry doesn't have `deadline` anyway.

Reply:
It seems "Slow tests take 30+ seconds" use `xdist: 2`.

Reply:
I measured and the test takes 30+ seconds.